src/Avalonia.Build.Tasks/XamlCompilerTaskExecutor.cs

```diff
@@ -89,6 +85,7 @@ MessageImportance logImportance
             if (debuggerLaunch)
             {
                 System.Diagnostics.Debugger.Launch();
+                while(!Debugger.IsAttached) Thread.Sleep(100);
```

Comment:
I was talking about a _separate_ mode for non-Windows/non-VisualStudio users. `Debugger.Launch()`, while being really convenient and useful, only works for VS on Windows

Reply:
I read [this](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debugger.launch?view=net-6.0#remarks) that Debugger.Launch() do nothing on Unix.

Reply:
Highly strange that this methods returns TRUE on Linux instead of FALSE.